src/Compilers/Core/Portable/CaseInsensitiveComparison.cs

```diff
@@ -183,10 +183,7 @@ public override int GetHashCode(string str)
         /// <summary>
         /// Returns a StringComparer that compares strings according the VB identifier comparison rules.
         /// </summary>
-        public static StringComparer Comparer
-        {
-            get { return s_comparer; }
-        }
+        public static StringComparer Comparer => s_comparer;
```

Comment:
@gafter I think he means that we should just use the get-only autoprop syntax here since the backing field is `readonly`. In fact, The get-only autoprop syntax is sugar for this exact pattern.